src/Main.java

```diff
@@ -0,0 +1,76 @@
+import java.util.*;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+class Main {
+    public static void main(String[] args) {
+
+
+        ArrayList<Student> students = new ArrayList<>();
+
+        Student student1 = new Student("Adam", "Małysz", 1L, Gender.male, 21, new Faculty("Electronics"), new ArrayList<Course>());
+        student1.addCourse(new Course("Maths"));
+        student1.addCourse(new Course("Physics"));
+        student1.addCourse(new Course("Biology"));
+        Student student2 = new Student("Zosia", "Samosia", 2L, Gender.female, 19, new Faculty("Architecture"), new ArrayList<Course>());
```

Comment:
Just to make it look nicer (formatting) - you could add an empty line after creation of each student (or move to the private helper method)